app/templates/components/learnergroup-members.hbs

```diff
@@ -1,18 +1,59 @@
 {{#liquid-if saving class='crossFade'}}
   {{wait-saving}}
 {{/liquid-if}}
-{{#each proxiedMembers as |userProxy|}}
-  <div class="form-col-12">
-    <label class="form-label learnergroup-username">{{userProxy.fullName}}</label>
-    <div class="form-data learnergroup-group-membership">
-      {{inplace-select
-        value=userProxy.lowestGroupInTree.id
-        options=learnerGroupOptions
-        save='changeLearnerGroup'
-        condition=userProxy.id
-        displayValueOverride=userProxy.groupDisplayValueString
-        selectPromptTranslation='learnerGroups.selectGroup'
-      }}
+
+{{#if multiEditModeOn}}
+  <div class="multi-edit__container">
+    <div class="multi-edit__box">
+      <label>{{t "learnerGroups.includeAll"}}</label>
+      {{input type="checkbox" checked=includeAll}}
     </div>
   </div>
-{{/each}}
+{{/if}}
+
+<div class="learnergroup-list">
+  {{#each proxiedMembers as |userProxy|}}
+    <div class="form-col-12">
+      <label class="form-label learnergroup-username">{{userProxy.fullName}}</label>
+      <div class="form-data learnergroup-group-membership">
+        {{inplace-select
+          value=userProxy.lowestGroupInTree.id
+          options=learnerGroupOptions
+          save='changeLearnerGroup'
+          condition=userProxy.id
+          displayValueOverride=userProxy.groupDisplayValueString
+          selectPromptTranslation='learnerGroups.selectGroup'
+          multiEditModeOn=multiEditModeOn
+          includeAll=includeAll
+          addStudent='addStudent'
+          removeStudent='removeStudent'
+        }}
+      </div>
+    </div>
+  {{/each}}
+</div>
+
+{{#if multiEditModeOn}}
+  {{#if errorMessage}}
+    <p class="continue-error-message">{{errorMessage}}</p>
+  {{/if}}
+
+  <button class="continue-button" {{action 'continueButton'}}>{{t 'general.continue'}}</button>
+{{/if}}
+
+{{#if editorModeOn}}
+  <div class="choose-action__container">
+    <div class="choose-action__options">
+      {{#ui-select field-component="learner-select-field" showPlaceholderAsOption=false placeholder='general.select' on-select="changeSelection" value=value inline=true items=proxiedOptions as |item|}}
+        {{#ui-option value=item}}
+          {{item.label}}
+        {{/ui-option}}
+      {{/ui-select}}
+    </div>
+
+    <span class='choose-action__actions'>
```

Comment:
Why make this a two step process?  Could just show the select menu and have a save button instead of going through the continue step?